sdk/kubernetesconfiguration/Microsoft.Azure.Management.KubernetesConfiguration/src/Properties/AssemblyInfo.cs

```diff
@@ -4,8 +4,8 @@
 using System.Reflection;
 using System.Resources;
 
-[assembly: AssemblyTitle("Microsoft Azure KubernetesConfiguration Management Library")]
-[assembly: AssemblyDescription("Provides Microsoft Azure KubernetesConfiguration management operations including the ability to create, update and delete runbooks and schedules.")]
+[assembly: AssemblyTitle("Microsoft Azure KubernetesConfiguration SourceControlConfigurations Management Library")]
+[assembly: AssemblyDescription("Provides Microsoft Azure KubernetesConfiguration SourceControlConfigurations management operations including the ability to create, update and delete configurations.")]
```

Comment:
Suggested changes as this package also contains KubernetesConfiguration Extension.
```suggestion
[assembly: AssemblyTitle("Microsoft Azure KubernetesConfiguration Management Library")]
[assembly: AssemblyDescription("Provides Microsoft Azure KubernetesConfiguration management operations including the ability to create, update and delete configurations and extensions.")]
```

Reply:
Updated in latest commit, PTAL